src/main/java/net/glowstone/entity/monster/GlowBoss.java

```diff
@@ -10,11 +12,34 @@
 
 /**
  * A monster with a boss bar.
+ *
+ * @author Chris Hennick
```

Comment:
While you did create that class, we don't use the `@author` tag in this project.

Reply:
Well there still are classes with that tag, but only for historical reasons.

Reply:
Reverted.